CHANGELOG.md

```diff
@@ -25,6 +24,8 @@
   - Enable return-await rule
   - Enable prefer-reduce-type-parameter rule
   - Enable prefer-readonly rule
+  - Enable no-inferrable-types rule
+  - Enable dot-notation rule
```

Comment:
were those changes to the changelog intended?

Reply:
Effect of conflict resolve. This section was in different line, I just moved them to one section.